src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -60,16 +61,22 @@ class AddressPage extends Component {
         this.onCountryUpdate = this.onCountryUpdate.bind(this);
 
         const currentCountry = lodashGet(props.privatePersonalDetails, 'address.country') || '';
+        const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_VALIDATION_REGEX, `${currentCountry}.zipFormat`, '');
```

Comment:
NAB - you don't "need" to concat this string, I'm pretty sure you can do this, which I think looks better:

```suggestion
        const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_VALIDATION_REGEX, [currentCountry, 'zipFormat'], '');
```